aperturedb/cli/ingest.py

```diff
@@ -25,6 +30,70 @@
 IngestType = Enum('IngestType', {k: str(k) for k in ObjectType._member_names_})
 
 
+def _debug_samples(data, sample_count, module_name):
+    import json
+    print(f"len(data)={len(data)}")
+    for i, r in enumerate(data[0:sample_count]):
+        q, b = r
+        with open(module_name + f"_{i}" + ".json", "w") as f:
+            f.write(json.dumps(q, indent=2))
+        for j, blob in enumerate(b):
+            with open(module_name + f"_{i}_{j}" + ".jpg", "wb") as f:
+                f.write(blob)
+
+
+def _apply_pipeline(data):
+    data = CommonProperties(data)
+    data = ImageProperties(data)
+    return data
+
+
+@app.command()
+def from_generator(filepath: Annotated[str, typer.Argument(
+    help="Path to python module for ingestion [BETA]")],
+    sample_count: Annotated[int, typer.Option(
+        help="Number of samples to ingest")] = 1,
+    debug: Annotated[bool, typer.Option(
+        help="Debug mode")] = False,
+    batchsize: Annotated[int, typer.Option(
+        help="Size of the batch")] = 1,
+    num_workers: Annotated[int, typer.Option(
+        help="Number of workers for ingestion")] = 1,
+    apply_image_properties: Annotated[bool, typer.Option(
+        help="Apply image properties to the AddImage command")] = True,
+):
+    """
+    Ingest data from a Data generator [BETA].
+    """
+    db = create_connector()
+    loader = ParallelLoader(db)
+
+    # not sure if this is the best way to do this.
+    # Hence the BETA tag.
+    module_name = os.path.basename(filepath)[:-3]
+    spec = importlib.util.spec_from_file_location(module_name, filepath)
+    module = importlib.util.module_from_spec(spec)
+    sys.modules[module_name] = module
```

Comment:
I think importlib is the defacto way to load modules dynamically. Why not explain explicitly what you're doing here